test/arrets_SUITE.erl

```diff
@@ -132,7 +132,7 @@ new_with_handle(doc) ->
   "exists";
 new_with_handle(_) ->
   Handle = arrets:new(handle),
-  ?assertNotEqual(undefined, ets:info(Handle)),
+  ?assertEqual(handle, ets:info(Handle)),
```

Comment:
`ets:info/1` does not return the handle ;)

But I'll fix this test in a separate commit